docs/documentation/release_notes/topics/24_0_0.adoc

```diff
@@ -1,3 +1,57 @@
+= Supported user profile and progressive profiling
+
+The user profile preview feature is promoted to be fully supported and user profile is enabled by default!
+
+In the past months, Keycloak team spent huge amount of effort in polishing user
+profile and making it fully supported now. In this release, we continued the effort started in the last release. Lots of improvements, fixes and
+polishing was done based on the thorought testing and feedback from our awesome community.
+
+Few highlights of the user profile feature:
+
+* Ability to specify what user attributes are managed and should be displayed on the forms to regular users or administrators
+
+* Dynamic forms - Previously, the forms where users created or updated their profiles, contain four basic attributes like username, email, first name and last name. Any additional
+attributes (or removing some of the default attributes) required to create custom theme. Now custom themes may not be needed anymore as users will see exactly the requested attributes based on the requirement
+of the particular deployment
+
+* Validations - Ability to specify validators for the user attributes. Lot of built-in validators allowing to specify maximum/minimum length, specified regex, fact
+that particular attribute must be URL or number and lots of other options
+
+* Annotations - Ability to specify that particular attribute should be rendered for instance as text area or HTML select with specified options or calendar and lots of other options
+
+* Progressive profiling - Allows to specify that some fields are required or available on the forms just for particular values of `scope` parameter. This effectively allow progressive
+profiling. No need anymore to ask user for twenty attributes during registration since you can instead ask user to fill attributes incrementally according to the requirements of individual client
+applications, which user want to use.
+
+* Migration from previous versions - User profile is now always enabled, but things should still work as before for those, who did not used user profile until now. You can
+benefit from user profile capabilities, but you are not strictly forced to use them. For migration instructions consult the link:{upgradingguide_link}[{upgradingguide_name}].
+
```

Comment:
This first release of the user profile as a supported feature is just the starting point and the baseline for delivering many more capabilities around identity management.